scalameta/tests/shared/src/test/scala/scala/meta/tests/CrossPlatformSemanticSuite.scala

```diff
@@ -10,7 +10,7 @@ class CrossPlatformSemanticSuite extends FunSuite {
   val classpath = Classpath(BuildInfo.mirrorClasspath)
 
   def semanticdbs: List[AbsolutePath] = classpath.deep.collect {
-    case path if v.Paths.isSemanticdb(path.name) =>
+    case path if v.SemanticdbPaths.isSemanticdb(path.name) =>
```

Comment:
These are always a little awkward due to the fact the meaning changes before and after the =>
Before the => this is a classpath, but after, if the check is correct, it is a semanticDbPath.  Even though it reads oddly, since the if is so close to the naming I tend to name it in the case that the off is correct, which makes readability of the code after the => better.
How about 
```
Case semanticDbPath if v.SemanticdbPaths.isSemanticdb(semanticDbPath.name) =>
```
At the point of checking in isSemanticdb(...) it is really candidatePath, it may or may not be true, but if the check succeeds this is definitely a semanticDbPath, not just any path so I prefer to at least carry that information along in the name since it is not in the type.